build.sbt

```diff
@@ -44,21 +45,20 @@ pomExtra := (
 
 isSnapshot := false
 
-publishTo :=
-  Some(if (isSnapshot.value) Opts.resolver.sonatypeSnapshots else Opts.resolver.sonatypeStaging)
+publishTo := sonatypePublishToBundle.value
```

Comment:
`sonatypePublishToBundle` & `sonatypeBundleRelease` are part of a faster release system implemented with `sbt-sonatype` v3.